src/util/token.ts

```diff
@@ -0,0 +1,9 @@
+import * as jwt from 'jsonwebtoken';
+// import * as dotenv from "dotenv";
+// dotenv.config();
+
+const jwtSecret = process.env.JWT_SECRET_KEY;
+
+export const generateToken : Function = ( email : Object ) : String => {
+    return jwt.sign(email, jwtSecret);
+}
```

Comment:
email 대신 userId 값을 넣어봤어요
```suggestion
export const generateToken : Function = ( userId: Int) : String => {
    return jwt.sign(String(userId), jwtSecret);
}
```